src/livecodes/templates/starter/zig-wasm-starter.ts

```diff
@@ -0,0 +1,93 @@
+import type { Template } from '../../models';
+
+export const zigWasmStarter: Template = {
+  name: 'zig-wasm',
+  title: 'Zig (Wasm) Starter',
+  thumbnail: 'assets/templates/zig.svg',
+  activeEditor: 'script',
+  markup: {
+    language: 'html',
+    content: `
+<div class="container">
+  <h1>Hello, <span id="name">World</span>!</h1>
+  <img class="logo" alt="logo" src="http://127.0.0.1:8080/livecodes/assets/templates/zig.svg" />
```

Comment:
We need to use baseUrl instead of absolute path for images in templates
```suggestion
  <img class="logo" alt="logo" src="{{ __livecodes_baseUrl__ }}assets/templates/zig.svg" />
```